apps/test/unit/gamelab/animationListModuleTest.js

```diff
@@ -82,4 +86,154 @@ describe('animationListModule', function () {
       expect(store.getState().animationList.propsByKey[key].sourceSize).not.to.be.undefined;
     });
   });
+
+  describe('action: set initial animationList', function () {
+    let oldWindowDashboard, server;
+    beforeEach(function () {
+      oldWindowDashboard = window.dashboard;
+      window.dashboard = {
+        project: {
+          getCurrentId() {return '';}
+        }
+      };
+
+      server = sinon.fakeServer.create();
+      server.respondWith('imageBody');
+    });
+
+    afterEach(function () {
+      server.restore();
+      window.dashboard = oldWindowDashboard;
```

Comment:
No need to do it in this PR, but since we replace window.dashboard in a few tests (and even in a [well-used test utility](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/util/testUtils.js#L11)) it'd be nice to extract a fakeDashboard helper that makes this easy to do in other tests.  I imagine a syntax modeled on `sinon`'s stubs and fakeServer:

```javascript
import {fakeDashboard} from '../testUtils';

let dashboard;
beforeEach(function () {
  dashboard = fakeDashboard();
});

afterEach(function () {
  dashboard.restore();
});
```